lib/insights/api/common/status.rb

```diff
@@ -0,0 +1,17 @@
+module Insights
+  module API
+    module Common
+      module Status
+        module Api
+          def health
+            if PG::Connection.ping(ENV['DATABASE_URL']) == PG::Connection::PQPING_OK
+              head :ok
+            else
+              head :internal_server_error
```

Comment:
Won't this cause the app pod to be killed and redeployed if the database is down?

Reply:
Yes, but is that a problem? Also I think that is going to be the least likely failure case.

Reply:
Restarting the pod when the database is down is not going to fix the problem

Reply:
But there could be other potential issues that cause us to be unable to connect to the database which would be solved by a restart. Maybe another thread is in some loop that built up too many connections to the database, maybe the certs changed and we need a reboot to pick up the new ssl config map or even the creds could have changed and the pod isn't restarted when the secret is updated.

These all seem more likely than RDS actually going down. Additionally, if the database is down, then restarting the pod doesn't really seem worse than every request failing and the pod being up. Every request is failing either way.

Reply:
Or maybe we hit some mysterious routing issue in openshift that prevents us from accessing a database which is up. (Not that that has ever happened :trollface:)

Reply:
I'm all for failing _very_ loudly when something else is down, especially since we'd be throwing 500s anyway.